control/trapezoid_velocity_profile.go

```diff
@@ -122,6 +122,10 @@ func (s *trapezoidVelocityGenerator) reset() error {
 	if s.kppGain == 0 {
 		s.kppGain = 0.45
 	}
+	s.lastVelCmd = 0
+	if s.cfg.Attribute.Has("init_vel") {
+		s.lastVelCmd = s.cfg.Attribute["init_vel"].(float64)
+	}
```

Comment:
another option is to make this a boolean to remember the speed.

Reply:
This is good enough for now - but I feel like it's a band-aid, we should be able to interrupt trapezoidal profiles with new speed asks, it's a pretty standard feedback control feature to rev up to a new set point.

Reply:
Also, we could get a zero force controller out of this package two years ago, so I think something regressed in its functionality, I can't wait for rover QA.

Reply:
Hmmm, I think like we should be able to pass the old state while the loop was paused through the `Signal` to `lastvelcommand`. This might not be the right place to deal with this, but I'm okay with a follow up ticket.

Reply:
I think I'm just going to tweak the code to explicitly do this, rather than having the roundabout configuration steps